ludwig/schema/metadata/parameter_metadata.py

```diff
@@ -29,6 +29,12 @@ class ExpectedImpact(int, Enum):
 class ParameterMetadata:
     """Contains descriptive information that pertains to a Ludwig configuration parameter."""
 
+    # Quick description generally for UI display
+    short_description: str = ""
+
+    # In depth description generally for documentation purposes.
```

Comment:
Does long_description support markdown syntax?

If so, we should make that clear in the doc string here.

If not, then I recommend removing the ` characters and other markdown things for readability.

Reply:
Since we currently aren't using the long descriptions, I think we can keep them. Down the line, I plan on using the long descriptions in a markdown friendly environment so I think it's best we keep them for now!